package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "github-notifier-chrome",
+  "version": "2.0.0",
+  "description": "> Chrome/Opera extension - Displays your GitHub notifications unread count",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint extension/*.js",
+    "test": "mocha"
+  },
+  "pre-commit": [
+    "lint",
+    "test"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sindresorhus/github-notifier-chrome.git"
+  },
+  "keywords": [
+    "github",
+    "notifier",
+    "chrome"
+  ],
+  "author": "Sindre Sorhus",
```

Comment:
Follow this format: https://github.com/sindresorhus/time-grunt/blob/master/package.json#L18.